src/translate.rs

```diff
@@ -157,7 +163,7 @@ impl <'a> ToGlibPtr<'a, *const c_char> for str {
 impl <'a> ToGlibPtr<'a, *const c_char> for String {
     type Storage = CString;
 
-    fn borrow_to_glib(&'a self) -> Stash<*const c_char, String> {
+    fn to_glib_none(&self) -> Stash<'a, *const c_char, String> {
         let tmp = CString::new(&self[..]).unwrap();
```

Comment:
`String` is a simple case, it's converting from references to objects that gave me a headache. And for a long time I didn't know how to avoid this lifetime here.